config/routes.rb

```diff
@@ -0,0 +1,12 @@
+Rails.application.routes.draw do
+  root "posts#index"
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :user, only: [:new, :create]
+  resources :sessions, only: [:new, :create, :destroy]
+  get '/signup', to: 'user#new'
+  post '/signup', to: 'user#create'
+
+  match '/signin',  to: 'sessions#new', via: 'get'
```

Comment:
The same thing as users sign up. If there is a resource then no need this way to create a route. So remove one of them. Either `resources :sessions` part or those match.

Reply:
Dipto Please see comments in the routes.rb about the sessions issues
Leon